tests/scripts/thread-cert/Cert_5_1_04_RouterAddressReallocation.py

```diff
@@ -32,26 +32,52 @@
 import mle
 import network_layer
 import thread_cert
+from pktverify.consts import MLE_ADVERTISEMENT, MLE_PARENT_REQUEST, MLE_PARENT_RESPONSE, MLE_CHILD_ID_RESPONSE, ADDR_SOL_URI, MODE_TLV, CHALLENGE_TLV, RESPONSE_TLV, LINK_LAYER_FRAME_COUNTER_TLV, CONNECTIVITY_TLV, ROUTE64_TLV, ADDRESS16_TLV, LEADER_DATA_TLV, NETWORK_DATA_TLV, SCAN_MASK_TLV, VERSION_TLV, LINK_MARGIN_TLV, SOURCE_ADDRESS_TLV, COAP_CODE_ACK, NL_STATUS_TLV, NL_RLOC16_TLV, NL_ROUTER_MASK_TLV
+from pktverify.packet_verifier import PacketVerifier
 
 LEADER = 1
 ROUTER1 = 2
 ROUTER2 = 3
 
+# Test Purpose and Description:
+# -----------------------------
+# The purpose of this test case is to verify that when the original Leader is
+# removed from the network, the DUT will create a new partition as Leader and
+# will assign a router ID if a specific ID is requested.
+#
+# Test Topology:
+# -------------
+#        Leader
+#       /     \
+# Router_1  -  Router_2
+# [DUT]
+#
+# DUT Types:
+# ----------
+# Router
+
 
 class Cert_5_1_04_RouterAddressReallocation(thread_cert.TestCase):
+    USE_MESSAGE_FACTORY = False
+    SUPPORT_NCP = False
```

Comment:
why not support NCP anymore?

Reply:
Leader needs to set partition_id = 1, but spinel-cli does not support this, so I disable ncp test for this test case

Reply:
fine to me.